src/import/tls/download_and_compile_mbedtls.sh

```diff
@@ -1,11 +1,27 @@
 #!/bin/bash
-
+echo
+echo "---------------"
+echo "mbedTLS LICENSE"
+echo "---------------"
+echo "Unless specifically indicated otherwise in a file, files are licensed"
+echo "under the Apache 2.0 license, as can be found in: apache-2.0.txt"
+echo
+echo "the apache-2.0.txt file can be accessed here:"
+echo "   https://github.com/ARMmbed/mbedtls/blob/development/apache-2.0.txt"
+echo
+read -p "Do you agree to the license of this third party library? (Y) " -n 1 -r
```

Comment:
Here is a line from ubuntu apt-get install question:

Do you want to continue? [Y/n] y

it has [] brackets and additionally they offer another option besides the Yes :) Isn't just offering a single option to pushful? (is this the correct word?